randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -460,7 +460,7 @@ Extra - room_id: [13, 46]
 > Door to Ripper Tower; Heals? False
   * Layers: default
   * Open Hatch to Ripper Tower/Door to Central Shaft
-  * Extra - door_idx: (31,)
+  * Extra - door_idx: (113,)
```

Comment:
history says that at one point, it was
![grafik](https://github.com/user-attachments/assets/655e52f0-05d4-4e66-b59c-2c4d4b181ad6)
 this:

Reply:
that's from combining the room states. Since the other room isn't used is there any point to add it back or just raising the comment for history sake

Reply:
forgot that we didnt use the other room anymore.

Reply:
![image](https://github.com/user-attachments/assets/3905d73d-3a15-4a79-8cf1-524fbd200558)
the door doesn't even exist in this state